### DISCUSSION TITLE
does codesandbox use `npm install` or `yarn add`?

### DISCUSSION THREAD

**Discussion by wolverineks** [codesandbox/codesandbox-client#4489] [General]
im getting a weird error, and it might be due to some differences in npm/yarn

```
Could not find module in path: 'readable-stream/errors' relative to '/node_modules/readable-stream/lib/internal/streams/state.js'
```

there might be some selective versioning going on, im not sure yet

**Comment by garethx** [comment]
Please provide an URL to a sandbox with this issue. Thanks.

**Reply by wolverineks** [reply]
https://codesandbox.io/s/edge-react-hooks-tesst-en2ib?file=/src/App.tsx
